hack/test-end-to-end-docker.sh

```diff
@@ -30,7 +30,17 @@ if [[ -z "${BASETMPDIR-}" ]]; then
   sudo rm -rf "${BASETMPDIR}"
   mkdir -p "${BASETMPDIR}"
 fi
-VOLUME_DIR="${BASETMPDIR}/volumes"
+
+# when selinux is enforcing, the volume dir selinux label needs to be
+# svirt_sandbox_file_t
+#
+# TODO: fix the selinux policy to either allow openshift_var_lib_dir_t
+# or to default the volume dir to svirt_sandbox_file_t.
+sudo mkdir -p /var/lib/openshift/openshift.local.volumes
```

Comment:
@pmorie is there anything we can do to detect they've configured VOLUME_DIR to something that won't work (selinux-wise) ahead of time, ideally at server start?